gtsfm/frontend/inlier_support_processor.py

```diff
@@ -75,7 +78,8 @@ def run(
             i2Ri1 = None
             i2Ui1 = None
             v_corr_idxs = np.array([], dtype=np.uint64)
-            return i2Ri1, i2Ui1, v_corr_idxs, None
+
```

Comment:
@ayushbaid @travisdriver @akshay-krishnan This computation here will give us the wrong numbers about inlier stats. If for 3 pairs, we had # inliers as 20, 20, 5, and if we threw away the 3rd, then we should have avg # inliers = 20.

But here it would be return as 15, which isn't correct.